tests/evm/test_extcodecopy.py

```diff
@@ -0,0 +1,183 @@
+import pytest
+from itertools import chain
+from zkevm_specs.copy_circuit import verify_copy_table
+from zkevm_specs.evm import (
+    verify_steps,
+    Tables,
+    Block,
+    Bytecode,
+    ExecutionState,
+    StepState,
+    Opcode,
+    U64,
+    U160,
+    RLC,
+    CopyCircuit,
+    CopyDataTypeTag,
+    AccountFieldTag,
+    CallContextFieldTag,
+)
+from zkevm_specs.evm.typing import RWDictionary
+from zkevm_specs.util import (
+    rand_fq,
+    keccak256,
+    EXTRA_GAS_COST_ACCOUNT_COLD_ACCESS,
+    MEMORY_EXPANSION_LINEAR_COEFF,
+    MEMORY_EXPANSION_QUAD_DENOMINATOR,
+    GAS_COST_COPY,
+)
+
+TESTING_DATA = (
+    (bytes(), True, True, 0x30000, 0x00, 0x00, 54),  # warm account with empty code
+    (bytes(), False, True, 0x30000, 0x00, 0x00, 54),  # cold account with empty code
+    (bytes([10, 40]), True, True, 0x30000, 0x00, 0x00, 54),  # warm account
+    (bytes([10, 10]), False, True, 0x30000, 0x00, 0x00, 54),  # cold account
+)
+
+
+def to_word_size(addr: int) -> int:
```

Comment:
For `to_word_size`, `memory_gas_cost`, and so on, you can use the helper methods from the `util` module here:
https://github.com/privacy-scaling-explorations/zkevm-specs/blob/master/src/zkevm_specs/util/testing.py

Reply:
Thanks! Fixed in [44faa4b](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/257/commits/44faa4b2ff67ca3d56fc362b3162f990a0f8ab59) :)